internal/plugin.go

```diff
@@ -352,6 +352,13 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 		p.PolicyDefaults.ConsolidateManifests = true
 	}
 
+	disabled, setDisabled := getDefaultBool(unmarshaledConfig, "disabled")
```

Comment:
This section isn't needed because the default value is `false` and the unmarshalled value of `p.PolicyDefaults.Disabled` will default to `false`, so we can just rely on that here.

This code would only be required if we default something to `true` but we want to detect if the user just omitted the value or actually explicitly set it to `false`.